examples/sizes.txt

```diff
@@ -26,6 +26,7 @@ chained_assignment                   83      83      83
 conditional_execution                380     344     344     
 conditional_expressions              229     181     181     
 contains                             155     156     156     
+control_op_simplification            44      48      48      
```

Comment:
@achidlow Note the increase in size here, if the optimisation is disabled, then the size goes down to 40 rather than up. However it seems the increase is due to additional copies introduced during SSA destructuring.

Reply:
Hmmm, might need to look at this in another PR.